packages/roosterjs-editor-types/lib/interface/FormatState.ts

```diff
@@ -174,6 +174,11 @@ export interface StyleBasedFormatState {
      * Direction of the element ('ltr' or 'rtl')
      */
     direction?: string;
+
+    /**
+     * Font weight
+     */
+    fontWeight?: string;

```

Comment:
Can you help me understand why we need this? We already have isBold in FormatState.

Reply:
it is a request for a partner team.

Reply:
They need semibold font-weight.

Reply:
I see. Since we modified FormatState and it is also shared with Content Model editor, we also need to let content model version of getFormatState returns this value. Otherwise other people see there is a fontWeight defined in FormatState but it never has value from Content model.